db-scheduler-ui-frontend/src/components/history/LogList.tsx

```diff
@@ -67,7 +68,12 @@ export const LogList: React.FC = () => {
       setSearchTermTaskName(taskName || '');
       setSearchTermTaskInstance(taskInstance || '');
     }
-  }, []);
+  }, [
```

Comment:
Er du sikker på at dette gjør det du vil? [] betyr jo kjør en gang for en useEffect, mens nå kjører den hver gang det her endrer seg